presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSessionProperties.java

```diff
@@ -335,14 +321,24 @@ public static DataSize getParquetMaxReadBlockSize(ConnectorSession session)
         return session.getProperty(PARQUET_MAX_READ_BLOCK_SIZE, DataSize.class);
     }
 
+    public static boolean isParquetBatchReaderVerificationEnabled(ConnectorSession session)
```

Comment:
@agrawalreetika These are the same as the ones in Hive. Can you please move the common ones to presto-hive-common/HiveCommonSessionProperties?

Reply:
+1, these are good candidates to be refactored and added to HiveCommonSessionProperties.